client/client.go

```diff
@@ -300,7 +298,7 @@ func (r *NotaryRepository) GetTargetByName(name string) (*Target, error) {
 
 // Publish pushes the local changes in signed material to the remote notary-server
 // Conceptually it performs an operation similar to a `git rebase`
-func (r *NotaryRepository) Publish(getPass passwordRetriever) error {
+func (r *NotaryRepository) Publish(getPass trustmanager.PassphraseRetriever) error {
```

Comment:
If the keyStoreManager already has the PassphraseRetriever, do we need to pass it in here?